task/buildah-rhtap/OWNERS

```diff
@@ -0,0 +1 @@
+RHTAP Build Team
```

Comment:
I think this should be a github team: https://github.com/orgs/redhat-appstudio/teams

Reply:
Do you think that adding the full url to the github team is enough?

Reply:
Just the name probably? I thought these were the Kubernetes OWNERS files, but looking at the docs our syntax is not correct so IDK https://www.kubernetes.dev/docs/guide/owners/

Reply:
Well, looking at the other tasks, they just use an internal team name. I don't think we need to set a new standard in this PR. I'll leave a note for this to be improved later.

`Konflux Build Team` seems to be what it is for now, so that's what I picked, but maybe we can stick to `Stonesoup Build Team` and change it the same time the other tasks do. :shrug:

Reply:
This is out of scope of this PR, let's just have a value for now and align all later.